cequence/README.md

```diff
@@ -0,0 +1,58 @@
+# Cequence
+
+## Overview
+
+This integration provides Cequence Security customers with a preconfigured dashboard to view all of their API Runtime Inventory and Threat Detection data within Datadog. This integration provides a Dashboard layout to provide in depth insights into your API Security posture. The log data export configuration must take place within your [Cequence Security Unified Platform.][3]
+
+## Setup
+
+All configuration is automated by this Datadog Integration. All customers need to do is click +Install to install the Cequence Datadog Dashboard and all of the necessary components will be deployed in Datadog.
+
+### Installation
+
+Navigate to Datadog Integrations and select Cequence. Then select Install.
+
+### Cequence Configuration
+
+1. Generate a new Integration API Key in your [Datadog portal][4].
+2. Use the following integration [article][3] to configure your Cequence Platform to Export Logs to Datadog 
+ - Be sure to configure your Data Export to use the following datadog API ingest url. This is critical for the functionality of this integration
+   https://http-intake.logs.datadoghq.com/api/v2/logs?ddsource=cequence
+
+
+### Datadog Configuration
+
+The following configuration must take place within the Datadog Portal.
+
+1. Login to your Datadog portal.
+2. Navigate to [Integrations][7]
+3. Search for Cequence
+4. Highlight the integration and Click + Install.
```

Comment:
```suggestion
```

Reply:
Suggesting to move these steps into the main setup section so that setup information can exist in one place.